defaults/main.yml

```diff
@@ -38,7 +38,7 @@ k8s_purge_cert_manager: no
 # https://github.com/jetstack/cert-manager/releases
 k8s_cert_manager_chart_version: "v1.2.0"
 k8s_cert_manager_namespace: cert-manager
-
+k8s_cert_manager_ingress_class: ngnix
```

Comment:
Could this use the same variable name as https://github.com/caktus/ansible-role-django-k8s/pull/49? Setting one and forgetting to set the other could lead to much confusion...

Alternatively, perhaps a comment above both that the other variable should be set as well (assuming both roles are being used) might help forestall any debugging challenges.